src/lcmap_client/util.py

```diff
@@ -0,0 +1,31 @@
+from math import floor
+from osgeo import gdal
+
+def tile_point_to_proj_point(tile_x, tile_y, transform):
```

Comment:
Hrn, since there are two functions, one for each direction, my suggestion from above isn't a good one. Let's think about how to make this more readable for future generations ...